servers/src/test/java/tachyon/worker/block/BlockStoreMetaTest.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.block;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import com.google.common.io.Files;
+import com.google.common.primitives.Ints;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.util.io.BufferUtils;
+import tachyon.worker.block.io.BlockWriter;
+import tachyon.worker.block.io.LocalFileBlockWriter;
+import tachyon.worker.block.meta.StorageDir;
+import tachyon.worker.block.meta.StorageTier;
+import tachyon.worker.block.meta.TempBlockMeta;
+
+/**
+ * Unit tests for tachyon.worker.block.BlockStoreMeta
+ */
+public class BlockStoreMetaTest {
+  private static final long TEST_USER_ID = 33;
+  /** block size in Bytes for test */
+  private static final long TEST_BLOCK_SIZE = 200;
+  /** num of dirs created for test in all different tiers */
+  private static final long TEST_DIR_NUM = 3;
+  /** num of total committed blocks */
+  private static final long COMMITTED_BLOCKS_NUM = 8;
+  
+  private BlockMetadataManager mMetadataManager;
+  private BlockStoreMeta mBlockStoreMeta;
+  
+  @Rule
+  public TemporaryFolder mTestFolder = new TemporaryFolder();
+  
+  @Before
+  public void before() throws Exception {
+    TachyonConf tachyonConf = new TachyonConf();
+    // Setup a two-tier storage: MEM:{1000}, HDD{3000, 5000}, 
+    // where missed SSD tier's quota is zero
+    String mTachyonHome = mTestFolder.newFolder().getAbsolutePath();
+    tachyonConf.set(Constants.TACHYON_HOME, mTachyonHome);
+    tachyonConf.set(Constants.WORKER_MAX_TIERED_STORAGE_LEVEL, "2");
+    tachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_ALIAS_FORMAT, 0), "MEM");
+    tachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_PATH_FORMAT, 0),
+        mTachyonHome + "/ramdisk");
+    tachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_QUOTA_FORMAT, 0),
+        2000 + "");
```

Comment:
Quota setting without any unit (b, kb, mb, gb, ...) will use default unit 'b'(Bytes).
See common module `tachyon.util.FormatUtils#parseSpaceSize()` for more detail.